storage/perfschema/pfs_variable.cc

```diff
@@ -92,7 +92,7 @@ bool PFS_system_variable_cache::init_show_var_array(enum_var_type scope,
   m_query_scope = scope;
 
 #ifndef NDEBUG
-  extern mysql_mutex_t LOCK_plugin;
+  extern mysql_mutex_t LOCK_plugin [[maybe_unused]];
```

Comment:
PS code does not have [[maybe_unused]] attribute. Why do we need it for PXC?
It should be in WITH_WSREP block.